scripts/jenkins/Jenkinsfile

```diff
@@ -1,3 +1,4 @@
+#!/usr/bin/env groovy
```

Comment:
I'm just tacking this change in here so that syntax highlighting works in the file in `vi`.